src/api/app/queries/open_requests_with_project_as_target_finder.rb

```diff
@@ -0,0 +1,9 @@
+class OpenRequestsWithProjectAsTargetFinder < OpenRequestsFinder
+  def initialize(relation, project_name)
+    super
+  end
+
+  def requests_finder
+    @relation.where(target_project: @project_name)
```

Comment:
```suggestion
    @relation.where(bs_request_actions: { target_project: @project_name })
```

`@relation` is `BsRequest` and no `BsRequestAction`

Reply:
And as tests/specs don't fail it's also clear this path isn't tested. You should test it...